cupy/cuda/cupy_cudnn.h

```diff
@@ -189,9 +166,17 @@ cudnnStatus_t cudnnActivationBackward(...) {
 
 extern "C" {
 
+
+#if !defined(CUPY_NO_CUDA)
```

Comment:
This line means `#if !defined(CUPY_NO_CUDA) & (CUDNN_VERSION > 2000)`.
CuPy supports version 2000 and later.